flexget/plugins/input/regexp_parse.py

```diff
@@ -152,13 +154,19 @@ def isvalid(self, entry):
     @plugin.internet(log)
     def on_task_input(self, task, config):
         url = config['source']
+        encoding = config.get('encoding')
 
         # if it's a file open it and read into content (assume utf-8 encoding)
         if os.path.isfile(os.path.expanduser(url)):
-            content = codecs.open(url, 'r', encoding='utf-8').read()
+            if encoding is None:
```

Comment:
Should the encoding be set to `utf-8` if `encoding` is `None`?

Reply:
In my opinion, `encoding` should not be set at all if it's `None` (to preserve whatever defaults), but the patch tries to not break the current behavior (when encoding for open is always set to utf-8).